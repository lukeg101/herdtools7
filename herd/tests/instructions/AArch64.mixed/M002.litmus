AArch64 M002

(* Tests STR barrel shifters *)

{ uint64_t x; 0:x0=1; 0:x2=x; int64_t 0:x3=1;}
P0                        ;
   STR W0, [X2, X3, LSL#2];

exists (x=4294967296)
