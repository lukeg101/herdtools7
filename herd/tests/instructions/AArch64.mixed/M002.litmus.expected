Test M002 Allowed
States 1
x=0x100000000;
Ok
Witnesses
Positive: 1 Negative: 0
Condition exists (x=0x100000000)
Observation M002 Always 1 0
Hash=822a4c2b0005fec10225b46de30bd2fe

